src/components/QuickOpenModal.js

```diff
@@ -293,10 +293,6 @@ export class QuickOpenModal extends Component<Props, State> {
       }
 
       if (results) {
-        if (this.isShortcutQuery()) {
```

Comment:
this is same function is already called in `this.selectResultItem` below. just redundant